thanks/thanks.py

```diff
@@ -61,15 +53,29 @@ def _display_thanks(self):
                 horizontal_bar=' ',
                 vertical_bar=' ',
             ))
+            cprint(
+                ''.join([
+                    "If you see projects with ",
```

Comment:
This notice is lovely, and nicely viral in growing the packages which will have funding info.